systemtests/src/test/java/io/enmasse/systemtest/bases/TestBase.java

```diff
@@ -166,9 +166,9 @@ protected Endpoint getMessagingRoute(AddressSpace addressSpace) throws Exception
 
     protected String getOCConsoleRoute() {
         if (kubernetes.getOcpVersion() == 4) {
-            return String.format("https://console-openshift-console.%s", environment.kubernetesDomain());
+            return String.format("https://console-openshift-console.%s", environment.kubernetesDomain()).replaceAll("(?<!(http:|https:))[//]+", "/");
```

Comment:
I'm missing your intent here.  Doesn't the code currently just replace a slash, which must be proceeded with a http/https protocol part with a slash?

Reply:
nope it finds all double slash and replace with single slash (except double slash after http ot https)

Reply:
see log from job: 2019-10-15T15:26:56.127Z INFO  [OpenshiftWebPage] Opening openshift web page on route https://<ip>:8443/console

Reply:
I misread it as a positive backward non grouping match, rather than negative one.   Are you saying ` environment.kubernetesDomain()`can include slashes?

Separately, repeating the same character in a character class is superfluous.  i.e. you can change `[//]+` to `[/]+` without changing meaning.